diff --git a/buildsystem/compile-libvlc.sh b/buildsystem/compile-libvlc.sh
index d320e1b..e68c3c9 100755
--- a/buildsystem/compile-libvlc.sh
+++ b/buildsystem/compile-libvlc.sh
@@ -14,7 +14,7 @@ AVLC_MAKE_PREBUILT_CONTRIBS=0
 # used instead of building the contribs from source
 AVLC_USE_PREBUILT_CONTRIBS=0
 # JNI build can be disabled for testing/CI purpose
-AVLC_BUILD_JNI=1
+AVLC_BUILD_JNI=0
 while [ $# -gt 0 ]; do
     case $1 in
         help|--help)
