diff --git a/src/modules/bank.c b/src/modules/bank.c
index 87101cb74b..134478f0dd 100644
--- a/src/modules/bank.c
+++ b/src/modules/bank.c
@@ -581,7 +581,7 @@ static void AllocateAllPlugins (libvlc_int_t *p_this)
 
 #ifdef HAVE_FORCED_PLUGINS
     /* Windows Store Apps can not load external plugins with absolute paths. */
-    AllocatePluginPath (p_this, "plugins", mode);
+    AllocatePluginPath (p_this, ".\\", mode);
 #else
     /* Construct the special search path for system that have a relocatable
      * executable. Set it to <vlc path>/plugins. */
